dev/integration_tests/flutter_gallery/windows/runner/utils.cpp

```diff
@@ -52,7 +52,11 @@ std::string Utf8FromUtf16(const wchar_t* utf16_string) {
     return std::string();
   }
   std::string utf8_string;
-  utf8_string.resize(target_length);
+  try {
+    utf8_string.resize(target_length);
+  } catch (...) {
+    return std::string();
+  }
```

Comment:
@tgucio which exception are you protecting against here? I believe `resize` only throws two exceptions:
1.`length_error` when `target_length` is greater than `std::string::max_size`, but this is typically a _huge_ value. You can protect against it with a check if needed.
2. `bad_alloc` which we don't protect against anywhere.

Reply:
@cbracken The idea was handle both errors (it's a code review item). If alloc errors are not to be handled then indeed this can be turned into size check vs. max_size for the very unlikely event of having to handle 4GB long strings - let me change it.